src/index.js

```diff
@@ -1,43 +1,26 @@
 import './style.css';
+import createListItem from './createlist.js';
 
-const tasks = [
-  {
-    description: 'Go grocery shopping',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Finish homework',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Take out the trash',
-    completed: false,
-    index: 3,
-  },
-];
+let tasks = [];
 
-function createListItem(task) {
-  const listItem = document.createElement('li');
-  listItem.innerHTML = `
-    <div class="task-item">
-      <div class="task-item__checkbox">
-        <input type="checkbox" id="task-${task.index}" ${task.completed ? 'checked' : ''}>
-        <label for="task-${task.index}"></label>
-      </div>
-      <div class="task-item__description ${task.completed ? 'completed' : ''}">
-        ${task.description}
-      </div>
-      <div class="task-item__delete">
-        <button class="delete-button" data-index="${task.index}">
-          <i class="fas fa-trash-alt"></i>
-        </button>
-      </div>
-    </div>
-  `;
-  return listItem;
+// Retrieve tasks from local storage
+const storedTasks = localStorage.getItem('tasks');
+if (storedTasks) {
+  tasks = JSON.parse(storedTasks);
 }
+
+// Add new task function
+function addTask(description) {
+  const newTask = {
+    description,
+    completed: false, // set the completed property to false
```

Comment:
- [ ] Good job @Christelle-12, while your project is great I believe you have missed the requirement that a property index should be set to the value of the new array length, the first task should have `index: 1` instead of `0` (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
![image](https://user-images.githubusercontent.com/69077061/230705161-d276edd2-1869-4b49-b846-6e51f076fc9e.png)
- [ ] Also, when a task is deleted, we should update all remaining items' indexes, so they represent the current list order and are unique.
![image](https://user-images.githubusercontent.com/69077061/230705208-7ec91792-1331-4a39-a30b-28ed6a7377a4.png)
- [ ] The last one is, all changes to the To-Do List are expected to be saved in local storage. I see the change to the `edit` task is not getting saved. Please address this for a better user experience.